### ISSUE TITLE
The output PDB file contains errors in sulfur residues and HIS residues

### ISSUE THREAD

**Issue by ygzn12138** [baker-laboratory/RoseTTAFold-All-Atom#93]
Sulfur residues have a hydrogen atom disconnected from carbon, and HIS residues exhibit an incorrect conformation.

**Comment by HBioquant** [comment]
Yes. I also found this in schrodinger software. Maybe there's something wrong with HIS topology definition in the script chemical.py?

![image](https://github.com/baker-laboratory/RoseTTAFold-All-Atom/assets/68720788/25819590-c817-4f4b-9bbe-841a908082f9)

**Comment by ygzn12138** [comment]
> 是的。我也在薛定谔软件中找到了这一点。也许脚本 chemical.py 中的HIS拓扑定义有问题？
> 
> ![图像](https://private-user-images.githubusercontent.com/68720788/325080522-25819590-c817-4f4b-9bbe-841a908082f9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwMTIxMjksIm5iZiI6MTcxNDAxMTgyOSwicGF0aCI6Ii82ODcyMDc4OC8zMjUwODA1MjItMjU4MTk1OTAtYzgxNy00ZjRiLTliYmUtODQxYTkwODA4MmY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI1VDAyMjM0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiYTI4MjQyZGQwZDc5Nzg0NWE5MmNlOWUxOWY4MDEzZmM5YmNlYWNiNzRhNjllY2UyOWJkZDIzMzczZjhhZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Z6-cDhuQ2KHI2YqIKB57CumNJ6yTsEoQtTvW9CvK_rc)

请问你知道应该怎么修改吗？我现在只能用schrodinger去把断开的原子手动连起来，然后用minimization功能去修复错误的HIS结构。

**Comment by HBioquant** [comment]
That's what I did. I think that's enough. From personal predicted structures and evaluations so far, it can be showed that RFAA needs to be improved in structure prediction, such as peptide bond breaking in the loop region, Steric clashes of all atoms, etc. Therefore, the energy minimization of the force field is strongly recommended for post-processing.

**Comment by fglaser** [comment]
I also found the wrong geometry in His, exactly as shown above with ChimeraX. 
Is there a work around o update to solve this problem?

Thanks a lot,
Fabian
